.github/workflows/build-test-e2e.yml

```diff
@@ -68,37 +70,21 @@ jobs:
       id-token: write
       contents: read
     steps:
-      - uses: actions/checkout@v3
-      - name: Check colima and lima version
-        if: runner.os == 'macos'
-        run: |
-          # Get colima and lima versions
-          limactl --version
-          colima version
-      - name: Setup docker (missing on MacOS)
-        if: runner.os == 'macos'
-        run: |
-          brew install docker
-          colima start
+      - uses: actions/checkout@v4
       - name: Download build from GitHub Actions Artifacts
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
         with:
           name: ${{ env.BUILD_ARTIFACT_NAME }}
           path: ${{ env.BUILD_DIR }}
-      - name: Run SWC on Tomcat
-        run: |
-          docker run \
-            --name "${{ env.CONTAINER_NAME }}" \
-            -d --rm \
-            -p 8888:8080 \
-            -v "/$(pwd)/${{ env.BUILD_DIR }}/${{ env.BUILD_NAME }}:/usr/local/tomcat/webapps/ROOT.war" \
-            -v "/$(pwd)/e2e_workflow/settings.xml":/root/.m2/settings.xml \
-            tomcat:9.0
-      - uses: actions/setup-node@v3
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
+      - name: Run app containers
+        run: pnpm docker:start
+      - uses: actions/setup-node@v4
         with:
           node-version: ${{ env.NODE_VERSION }}
       - name: Install dependencies
-        run: pnpm install --frozen-lockfile --network-timeout 1000000
```

Comment:
safer to keep the (I assume increased) network timeout?

Reply:
This was a `yarn` command line argument, `pnpm` doesn't recognize it.